feature-utils/poly-look/src/react-components/infographic/infographic.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./infographic.css";
+
+/** Renders an image and a numbered legend that contains descriptions
```

Comment:
Minor nit: please use "indexed" instead of "numbered" so that it's crystal-clear you mean the `idx` in the code.